packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1141,10 +1182,21 @@ class Auth {
         })
 
         if (shouldReloginCurrentSession) {
-            await this.loginRegisteredUserB2C({
-                username: login,
-                password
-            })
+            const loginParams: LoginRegisteredUserB2CCredentials = {
+                slasClient: this.client,
+                credentials: {
+                    username: login,
+                    password
+                },
+                parameters: {
+                    redirectURI: this.redirectURI,
+                    dnt: this.getDnt({includeDefaults: true}),
+                    ...(this.get('usid') && {usid: this.get('usid')})
+                }
+            }
+
+            const token = await helpers.loginRegisteredUserB2C(loginParams)
+            this.handleTokenResponse(token, false)
```

Comment:
reuse this.this.loginRegisteredUserB2C because it takes ECOM clear session into account